modules/swagger-codegen/src/main/resources/Javascript/es6/api_test.mustache

```diff
@@ -1,15 +1,9 @@
 {{>licenseInfo}}
+import expect from 'expect';
+import {{moduleName}} from '../../src/index';
+
 (function(root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    // AMD.
-    define(['expect.js', '../../src/index'], factory);
-  } else if (typeof module === 'object' && module.exports) {
-    // CommonJS-like environments that support module.exports, like Node.
-    factory(require('expect.js'), require('../../src/index'));
-  } else {
-    // Browser globals (root is window)
-    factory(root.expect, root.{{moduleName}});
-  }
+  factory(expect, {{moduleName}});
```

Comment:
Does it still need to be ES5 style module (I mean the whole `(function(root, ...` seems to be redundant, but I am not 100% sure)?

Reply:
I am really not expert enough to be sure of that.

I can ask a friend to have a look if no one is sure...